package.json

```diff
@@ -63,6 +63,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-i18next": "^11.2.1",
+    "react-iframe": "^1.8.0",
```

Comment:
BTW, you probably don't need this package, you could probably just use the native `iframe` or `object` tag

Reply:
You're right, thanks! Replaced it in 063edce